insighthub-frontend/src/App.tsx

```diff
@@ -4,7 +4,10 @@ import Login from './pages/Login';
 import Register from './pages/Register';
 import './App.css'
 import Dashboard from './pages/Dashboard';
+import Logout from './pages/Logout';
 
+
+// maybe need to be like " <Route path="/logout" element={<Logout />}  />" ?
```

Comment:
Can you please remove this comment? It looks like it was implemented in line 19.